h2o-core/src/test/java/water/parser/ChunksizeTest.java

```diff
@@ -122,17 +127,17 @@ public void run() throws IOException {
     }
     fw.close();
       for (int i : new int[]{0,1}) {
-        Log.info((i==1 ? "Old" : "New") + " heuristic:");
-        Log.info("Total: " + counter[i]);
-        Log.info("Failure rate: " + PrettyPrint.formatPct((double) failed[i] / counter[i]));
-        Log.info("Too small: " + PrettyPrint.formatPct((double) toosmall[i] / counter[i]));
-        Log.info("Too large: " + PrettyPrint.formatPct((double) toolarge[i] / counter[i]));
-        Log.info("Too few: " + PrettyPrint.formatPct((double) toofew[i] / counter[i]));
-        Log.info("Too many: " + PrettyPrint.formatPct((double) toomany[i] / counter[i]));
+//        Log.info((i==1 ? "Old" : "New") + " heuristic:");
```

Comment:
See, this too has no informational value, and clutters the output greatly. It might have been good for testing, I don't know.